contribs/gmf/src/services/objecteditingquery.js

```diff
@@ -220,3 +220,17 @@ gmf.ObjectEditingQuery.GetQueryableLayerNodesResponse;
  * }}
  */
 gmf.ObjectEditingQuery.QueryableLayerInfo;
+
```

Comment:
Right.  But telling it to 'export' must ensure that both `a` and the original property exist.  The patch does fix the issue, right?